lib/mdl/rules.rb

```diff
@@ -797,84 +797,91 @@ def check_blockquote(errors, elements)
   end
 end
 
-rule 'MD055', 'Tables should be well-formed' do
+rule 'MD055', 'Table row doesn\'t begin/end with pipes' do
   tags :tables
-  aliases 'table-format-validation'
+  aliases 'table-rows-start-and-end-with-pipes'
   check do |doc|
     error_lines = []
     tables = doc.find_type_elements(:table)
     lines = doc.lines
-    num_lines = lines.length
 
     tables.each do |table|
-      loc = table.options[:location]-1
-      header = lines[loc]
+      table_pos = table.options[:location] - 1
+      table_rows = get_table_rows(lines, table_pos)
 
-      if header[0] != '|' || header[-1] != '|' || header.length == 1
-        # Tables should start and end with '|'
-        error_lines << (loc+1)
+      table_rows.each_with_index do |line, index|
+        if line.length == 1 || line[0] != '|' || line[-1] != '|'
```

Comment:
I know that 0 would not be recognized as a table, but just for clarity, should this be `line.length < 2`  since that's technically what we're looking for here?

Reply:
Yeah, we can do that.